parsers/binplist.py

```diff
@@ -164,7 +164,7 @@ def top_level_object(self):
     return self._ParseObjectByIndex(self.top_level_index, self.object_offsets)
 
   def Open(self, file_obj):
-    data = file_obj.read()
+    data = file_obj.read(-1)
```

Comment:
So during testing yesterday I realized that -1 doesn't behave like it does in other places for reading the full file.  I will be pushing a completed branch for this soon for review.